src/tokens/tokens.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+
+import src.common.func_pictures as fct
+from src.common.config import ConfigPath
+
+
+class Tokens:
+    data = dict()
+    
+    @classmethod
+    def initialize(cls, file):
+        with open(file) as f:
+            cls.data = json.load(f)
+        
+        # Convert key values as integer
+        # And load images
+        for k in list(cls.data.keys()):
```

Comment:
Don't need of `list()` ??

Reply:
`data` is modified as I run over it and `data.keys()` returns an iterator.
So I have to convert it as a list.